.github/workflows/check-release.yml

```
@@ -0,0 +1,35 @@
+name: Check Release
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [check_release, link_check]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Base Setup
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      - name: Install Dependencies
+        run: |
+          pip install -e .
+      - name: Check Release
+        if: ${{ matrix.group == 'check_release' }}
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Run Link Check
+        if: ${{ matrix.group == 'link_check' }}
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-links@v1
+      - name: Upload Distributions
+        uses: actions/upload-artifact@v2
+        with:
+          name: jupyter-releaser-dist-${{ github.run_number }}
```

Maybe with the `nbgrader` prefix to make it more explicit when downloading the artifact?

```suggestion
          name: nbgrader-jupyter-releaser-dist-${{ github.run_number }}
```